src/codeu/chat/common/User.java

```diff
@@ -51,12 +54,25 @@ public User read(InputStream in) throws IOException {
   public final Uuid id;
   public final String name;
   public final Time creation;
+  public HashSet<Uuid> interestedConvos = new HashSet<>();
+  public HashSet<Uuid> interestedUsers = new HashSet<>();
+  public Time lastUpdateConvos;
```

Comment:
I'd change these two Time variables to `private` since you're using public accessor methods to get their values, and it's good to keep other classes from manipulating this data unless necessary.